lib/logstash/outputs/sqs/patch.rb

```diff
@@ -0,0 +1,18 @@
+# This patch was stolen from logstash-plugins/logstash-output-s3#102.
```

Comment:
Just to add I wont block if the patch stays in the sqs output, if we move the patch to the mixin-aws we will need to adjust the gemspec to have a minimal version of the mixin that include that patch.